plasmapy/formulary/parameters.py

```diff
@@ -54,17 +56,11 @@
     "Reynolds_number",
     "Rm_",
 ):  # coverage: ignore
-    try:
+    with contextlib.suppress(ValueError):  # ValueError if name was not in __aliases__
         __aliases__.remove(name)
-    except ValueError:
-        # name was not in __aliases__
-        pass
 
-    try:
+    with contextlib.suppress(ValueError):  # ValueError if name was not in __all__
         __all__.remove(name)
```

Comment:
These could be the same block.

Reply:
I had wondered the same thing, but if there's an exception raised in the first line of the [`contextlib.suppress`](https://docs.python.org/3/library/contextlib.html#contextlib.suppress) block, the second line won't be run.  That would have been nice, though.

Reply:
Oh, huh, didn't know that! Then there really is no difference between the try block and the suppress call.